### ISSUE TITLE
What do you think about org-super-links?

### ISSUE THREAD

**Issue by Ypot** [branweb1/backlinks#1]
Thanks for your package. I will try it. By the way, it seems it could work well with org-super-links. For example, this issue seems to be something like what you made:
https://github.com/toshism/org-super-links/issues/13

**Comment by branweb1** [comment]
Hmmm the idea behind that package is that if I create a link from A to B, a backlink is automatically created from B to A? That would facilitate moving back and forth between A and B but that's not really what I wanted to do here. I was trying to implement this idea: https://notes.andymatuschak.org/z4KZ9973AoHhvM9Pj5Qrds48JXNbMEwVJmVRw. I want to go to a file and view it's backlinks to hopefully stumble upon a novel connection. I also want to auto-create a link target if it doesn't exist (have yet to implement this feature). The idea is that if A,B,and C link to a blank file D, that defines a kind of conceptual boundary that helps me to write D. So I guess the tl;dr is that I think this package is trying to do something functionally similar to super-links but not conceptually similar.

**Comment by Ypot** [comment]
Hi!
That https://notes.andymatuschak.org/z4KZ9973AoHhvM9Pj5Qrds48JXNbMEwVJmVRw it's exactly the idea. Maybe the misunderstanding is that we are working with linked headings and you seem to be working with linked files.

**Comment by branweb1** [comment]
yeah perhaps you're right. I'm currently thinking through this again. Working at the file-level seemed more intuitive to me at first, but now that I've worked with it, I think working at the headings-level is certainly more idiomatic to org-mode. Tagging and a bunch of other features somewhat insist on this, and it seems like a bad idea to swim upstream against them.
